dist/css/dress-code.css

```diff
@@ -1824,8 +1660,8 @@ body {
 
 @font-face {
   font-family: "DressCodeIcons";
-  src: url("../fonts/DressCodeIcons.eot");
-  src: url("../fonts/DressCodeIcons.eot?#iefix") format("eot"), url("../fonts/DressCodeIcons.woff") format("woff"), url("../fonts/DressCodeIcons.ttf") format("truetype"), url("../fonts/DressCodeIcons.svg#DressCodeIcons") format("svg");
+  src: url("../fonts/DressCodeIcons.eot?v=2.0.0-beta");
+  src: url("../fonts/DressCodeIcons.eot?#iefix") format("eot"), url("../fonts/DressCodeIcons.woff?v=2.0.0-beta") format("woff"), url("../fonts/DressCodeIcons.ttf?v=2.0.0-beta") format("truetype"), url("../fonts/DressCodeIcons.svg?v=2.0.0-beta#DressCodeIcons") format("svg");
 }
```

Comment:
@sambhav-gore Sorry my fault yes it's automated I miss the order of tasks for releasing.... good catch!